src/Features/Core/Portable/EditAndContinue/AbstractEditAndContinueAnalyzer.cs

```diff
@@ -1162,7 +1165,7 @@ public LambdaInfo WithMatch(Match<SyntaxNode> match, SyntaxNode newLambdaBody)
 
                     // exception handling around the statement:
                     CalculateExceptionRegionsAroundActiveStatement(
-                        bodyMatch,
+                        match,
```

Comment:
Could also have removed line 1150 where I set `match = bodyMatch` and instead made this `hasMatching ? match! : bodyMatch` or even `match ?? bodyMatch`. This way seemed a little more future proof.